### PR TITLE
libbitcoinconsensus: Add version field to pkg-config info file

### PR DESCRIPTION
**Pull request by rohythakur**

### CODE REVIEW

File: libbitcoinconsensus.pc.in [hunk 1]
```diff
@@ -2,6 +2,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
+Version=@VERSION@
```
Comment (promag) [inline libbitcoinconsensus.pc.in]: Usually version is below `Description: ...`.

**Review by promag — COMMENTED** [review]
Concept ACK.

File: libbitcoinconsensus.pc.in (hunk 1 above)
Comment (laanwj) [inline libbitcoinconsensus.pc.in]: Why is Version with a capital and the rest with lowercase, it that intentional?

**Comment by laanwj** [comment]
Please improve your commit message and PR title, just "add version" does not explain anything.

ping @theuni for review

File: libbitcoinconsensus.pc.in (hunk 1 above)
Comment (promag) [inline libbitcoinconsensus.pc.in]: From other examples yes.

**Comment by luke-jr** [comment]
Tested ACK, after commit message is fixed.

Suggest: `libbitcoinconsensus: Add version field to pkg-config info file`

**Comment by fanquake** [comment]
utACK after commit fix, updated the PR title to @luke-jrs suggestion.

**Comment by theuni** [comment]
I don't think this hurts anything, but adding the bitcoin core version here to query against seems counter to the purpose of the lib.

That said, the change itself is fine.

**Comment by laanwj** [comment]
> but adding the bitcoin core version here to query against seems counter to the purpose of the lib

Yes, I guess ideally it should be versioned based on API, not on bitcoin core version?

**Comment by laanwj** [comment]
I do think this still needs more discussion, how to version the bitcoin-consensus lib.

**Pull request closed without merging**